# net resources
endpoint.err.fatal=Punto Final (Endpoint) {0} parado debido a excepción: {1}
endpoint.err.nonfatal=El Punto Final (Endpoint) {0} ignoró excepción: {1}
endpoint.warn.reinit=Reinicializando ServerSocket
endpoint.warn.restart=Rearrancando punto final (endpoint)
endpoint.warn.security=Punto Final (Endpoint) {0} con excepción de seguridad: {1}
